src/components/StudioEntrypoint.tsx

```diff
@@ -15,36 +14,24 @@ type StudioEntrypointProps = {
 
 export const StudioEntrypoint: React.FC<StudioEntrypointProps> = ({ enterLocation }) => {
   const {
-    activeUser: { accountId, memberId, channelId },
-    setActiveChannel,
-  } = useActiveUser()
-  const { extensionConnected: extensionStatus, accounts } = useJoystream()
-
-  const { membership, loading: membershipLoading } = useMembership(
-    {
-      where: { id: memberId },
-    },
-    {
-      skip: !memberId,
-    }
-  )
-
-  const { memberships, loading: membershipsLoading } = useMemberships(
-    {
-      where: { controllerAccount_in: accounts.map((a) => a.id) },
-    },
-    {
-      skip: !accounts.length,
-    }
-  )
+    activeAccountId,
+    activeMemberId,
+    activeChannelId,
+    setActiveUser,
+    extensionConnected: extensionStatus,
```

Comment:
```suggestion
    extensionConnected
```
and we could just remove line 28.